src/it/resources/org/checkstyle/suppressionxpathfilter/avoidescapedunicodecharacters/SuppressionXpathRegressionAvoidEscapedUnicodeCharactersTwo.java

```diff
@@ -0,0 +1,5 @@
+package org.checkstyle.suppressionxpathfilter.avoidescapedunicodecharacters;
+
+public class SuppressionXpathRegressionAvoidEscapedUnicodeCharactersTwo {
+    private String unitAbbrev9 = "\u03bcs"; /* warn */
```

Comment:
please add here escape that is ignored by check (with property allowEscapesForControlCharacters)

Reply:
Okay

Reply:
Done